src/components/zen-avatar-details/zen-avatar-details.tsx

```diff
@@ -31,19 +31,22 @@ export class ZenAvatarDetails {
 
   @Element() host: HTMLZenAvatarDetailsElement;
 
-  /** Different variants  */
+  /** Set fifferent variants  */
```

Comment:
```suggestion
  /** Set the style variant  */
```

First I wanted to change the `fifferent` :P, but then I realized that the implementation doesn't set "variants", it chooses a single variant from a list.

Reply:
Good catch. And i like this fix suggestions!